(4b) Squads for VP/UI/SquadBases.lua

```diff
@@ -0,0 +1,102 @@
+print("Loading SquadBases.lua")
+
+-- third party
+include ( "TableSaverLoader.lua" )
+
+local squadBasesLut = {};
+
+function GetSquadBasePlot(squadNumber)
+    return squadBasesLut[squadNumber];
+end 
+
+function SetSquadBasePlot(squadNumber, newBasePlot)
+    squadBasesLut[squadNumber] = newBasePlot;
+end
+
+function ResetSquadBasePlot(squadNumber)
+    squadBasesLut[squadNumber] = nil
+end
+
+for i=1,10 do
```

Comment:
So there is a limit in 10 squads? What if there are more squads?

Reply:
There's a limit of 10 squads, yes, though this can be extended if people are finding this to be too small a number (I have not heard anyone say this in 2 years).
There should be an equivalent `LoadTable()` call after the local map is initialized though I think perhaps the whole squad bases thing should be removed, don't think anyone uses it  (if this indeed is broken and nobody has spoken up about it that's probably pretty indicative of noone even trying to use it lol)